occurrences/tests/test_event_api_services.py

```diff
@@ -15,15 +16,34 @@ def test_send_event_republish(
     mock_update_event_data,
     p_event,
 ):
-    OccurrenceFactory(p_event=p_event)
+    OccurrenceFactory.create_batch(3, p_event=p_event, cancelled=False)
+
+    first_occurrence = p_event.occurrences.order_by("start_time").first()
+    last_occurrence = p_event.occurrences.order_by("end_time").last()
```

Comment:
You could also use little more succinct versions `.earliest("start_time")` and `.latest("end_time")`.